source/community/reactnative/src/components/containers/column.js

```diff
@@ -201,8 +88,42 @@ export class Column extends React.Component {
 			return null;
 		}
 	}
+	/**
+     * @description This function calculates flex value
+	 * 				based on the column width property from the payload. 
+	 * @returns {flex}
+     */
+
+	flex = (containerViewStyle) => {
```

Comment:
Columns with explicit width is not working as expected. (Example: Refer`Column.Explicit.Size.json` in Visualizer - Though width for the column is mentioned as `50px`, the renderer doesn't respect the value)

Reply:
Resolved.